contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -2709,6 +2766,27 @@ class tsqlMutator : public TSqlParserBaseListener
 	{
 		in_procedure_parameter = false;
 		in_procedure_parameter_id = false;
+
+		// Look for parameter defaults that use an @@variable with no preceding whitespace
+		if (ctx->EQUAL())
+		{
+			// The '=' char must be followed immediately by the variable without any character in between
+			Assert(ctx->expression());			
+			size_t startPosition = ctx->expression()->start->getStartIndex();
+			if ((startPosition - ctx->EQUAL()->getSymbol()->getStopIndex()) == 1)
+			{
+				std::string var = getFullText(ctx->expression());
+				// The subsequent default expression must be a variable starting with '@@'
+				if (var.front() == '@') 
+				{
```

Comment:
Those two if condition can be combined into one

Reply:
see comment above